charts/sonarqube/README.md

```diff
@@ -82,6 +82,7 @@ The following table lists the configurable parameters of the Sonarqube chart and
 | `serviceAccount.create`             | If set to true, create a serviceAccount                                                                                   | false                                          |
 | `serviceAccount.name`               | Name of the serviceAccount to create/use                                                                                  | `sonarqube-sonarqube`                          |
 | `serviceAccount.annotations`        | Additional serviceAccount annotations                                                                                     | `{}`                                           |
+| `account.admin_password`            | Change default admin password                                                                                             | `"admin"`                                           |
```

Comment:
Please document all the new values under account.

Reply:
fix